### ISSUE TITLE
Fel connects but data not transferring

### ISSUE THREAD

**Issue by scheckley** [Thore-Krug/Flash-CHIP#17]
Running the script on Debian Buster:

```== preparing images ==
== Local/cached probe files located ==
== Staging for NAND probe ==
Image Name:   detect NAND
Created:      Sat Oct 26 19:58:32 2019
Image Type:   ARM Linux Script (uncompressed)
Data Size:    97 Bytes = 0.09 KiB = 0.00 MiB
Load Address: 00000000
Entry Point:  00000000
Contents:
   Image 0: 89 Bytes = 0.09 KiB = 0.00 MiB
waiting for fel....OK
usb_bulk_send() ERROR -7: Operation timed out
```
works ok on a Pi 3 running Jessie.

**Comment by Thore-Krug** [comment]
Did u run this on the same Pi ?

**Comment by scheckley** [comment]
no - laptop running MX Linux 19 (Buster) and a separate Rpi3 running OSMC. Still searching to see if it's a local issue on the laptop.

**Comment by Thore-Krug** [comment]
This error is power related so I might be a issue with your Laptop.

**Comment by scheckley** [comment]
ah ok, I will see if there's some kind of power saving on the usb port.  otherwise, the only difference I can see is sunxi-tools is version 1.4.1 on the Pi and 1.4.2 on the laptop. I tried using a usb-c port on the laptop via an adapter and that gave the same -7 error, so possibly it's the OS rather than a malfunctioning socket, but I'll continue to investigate.

**Comment by scheckley** [comment]
also investigating this https://github.com/linux-sunxi/sunxi-tools/issues/100

could be related to sunxi-tools version in buster

**Comment by 1sm1rk** [comment]
can confirm that, having the same problem with debian buster, did a downgrade of sunxi-tools to 1.4.1-1 from debian stretch. Works like a charm.

**Comment by thiemok** [comment]
I had the same issue, but using sunxi-tools 1.4.1-1 from stretch on ubuntu 19.10 did not solve it for me.
However I could get it to work by connecting an external power supply to the chip.

**Comment by Thore-Krug** [comment]
It's a Power related issue so yeah providing more power should fix it.

**Comment by thiemok** [comment]
Would it be worth to add this as a hint to the readme? Seems like people seem to miss this as a relatively simple fix when running into this problem.

**Comment by Thore-Krug** [comment]
I will add it later

**Comment by thiemok** [comment]
No worries I can quickly make a pr

**Comment by RM250J** [comment]
I had the same problem, tried all sorts of different cables on an Ubuntu Eoan install with no luck. I had no idea how to downgrade sunxi tools to 1.4.1, so I went the long route, created a Debian Stretch Virtual machine and it worked first time with the same cables

**Comment by sridharrajagopal** [comment]
I'm facing the same issue. One chip successfully flashed, but the other is giving this problem. I modified the scripts to add verbosity to sunxi-fel, and saw that this is the command that fails:

sudo sunxi-fel --verbose spl .new/firmware/images/sunxi-spl.bin
Enabling the L2 cache
Stack pointers: sp_irq=0x00002000, sp=0x00005DF8
Reading the MMU translation table from 0x00008000
Disabling I-cache, MMU and branch prediction... done.
=> Executing the SPL... done.
usb_bulk_send() ERROR -7: Operation timed out

I've tried attaching the chip to a powered USB hub, but still have the same issue. 

Thanks,
Sridhar
